### PR TITLE
docs: document how external link icons work

### PR DESCRIPTION
**Pull request by ajitsinghkaler**
After PR #36601 which added icons to all external links. Documented how this is happening via comments in scss file. For details visit PR #36601

@IgorMinar suggested that we should add documentation on how external link icons work. For details see https://github.com/angular/angular/pull/36601#issuecomment-625952724

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

No documentation on how external link icons work

Issue Number: N/A

## What is the new behavior?
No documentation on how external link icons work

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by ajitsinghkaler** [comment]
@IgorMinar can you please look into this

**Comment by ajitsinghkaler** [comment]
@IgorMinar can you please look into this

**Comment by ajitsinghkaler** [comment]
@IgorMinar can yo please look into this

File: aio/src/styles/0-base/_typography.scss [hunk 1]
```diff
@@ -193,6 +193,12 @@ code {
   }
 }
 
+// The following css adds an icon to external icons in the docs area. The following classes
+// .folder-api, .folder-cli, .folder-docs,.folder-guide, .folder-start, .folder-tutorial
+// contains all the documentation in the aio website.
```
Comment (IgorMinar) [inline aio/src/styles/0-base/_typography.scss:198]:
```suggestion
```

File: aio/src/styles/0-base/_typography.scss [hunk 2]
```diff
@@ -193,6 +193,12 @@ code {
   }
 }
 
+// The following css adds an icon to external icons in the docs area. The following classes
+// .folder-api, .folder-cli, .folder-docs,.folder-guide, .folder-start, .folder-tutorial
```
Comment (IgorMinar) [inline aio/src/styles/0-base/_typography.scss:197]:
```suggestion
```

File: aio/src/styles/0-base/_typography.scss [hunk 3]
```diff
@@ -193,6 +193,12 @@ code {
...
+// contains all the documentation in the aio website.
+// Added an icon in `:after` of all the links which contain http or https. These define all external links.
```
Comment (IgorMinar) [inline aio/src/styles/0-base/_typography.scss:199]:
```suggestion
// We add the icon to all external links which are identified as absolute links (those that start with `http` or https`).
```

File: aio/src/styles/0-base/_typography.scss [hunk 4]
```diff
@@ -193,6 +193,12 @@ code {
...
+// For Details visit PR #36601
```
Comment (IgorMinar) [inline aio/src/styles/0-base/_typography.scss:200]:
```suggestion
// For more info see PR #36601
```

File: aio/src/styles/0-base/_typography.scss [hunk 5]
```diff
@@ -213,6 +219,9 @@ code {
       }
     }
 
+    // Docs body contain links to github which are icons these are also external links
```
Comment (IgorMinar) [inline aio/src/styles/0-base/_typography.scss:222]:
```suggestion
    // The docs-viewer also contain links to GitHub (e.g. the edit this page icon) identified with `.github-links` class.
```

File: aio/src/styles/0-base/_typography.scss [hunk 6]
```diff
@@ -213,6 +219,9 @@ code {
...
+    // Don't want to add icon to these links. So icons are hidden on these links
```
Comment (IgorMinar) [inline aio/src/styles/0-base/_typography.scss:223]:
```suggestion
    // We don't want to add the external link icon to these links, so we hide them.
```

File: aio/src/styles/0-base/_typography.scss [hunk 7]
```diff
@@ -213,6 +219,9 @@ code {
       }
     }
 
+    // Docs body contain links to github which are icons these are also external links
+    // Don't want to add icon to these links. So icons are hidden on these links   
+    // For Details visit PR #36601
```
Comment (IgorMinar) [inline aio/src/styles/0-base/_typography.scss:224]:
```suggestion
```

**Review by IgorMinar — CHANGES_REQUESTED** [review]

**Review by IgorMinar — APPROVED** [review]
LGTM now. thanks!

**Pull request closed without merging**